backend/src/app/rpc/commands/files.clj

```diff
@@ -492,10 +492,13 @@
 
           (library-summary [{:keys [id data] :as file}]
             (binding [pmap/*load-fn* (partial load-pointer conn id)]
-              {:components (assets-sample (:components data) 4)
-               :media (assets-sample (:media data) 3)
-               :colors (assets-sample (:colors data) 3)
-               :typographies (assets-sample (:typographies data) 3)}))]
+              (let [components-sample (-> (assets-sample (:components data) 4)
+                                          (update :sample
+                                                  #(map (partial ctf/load-component-objects data) %)))]
```

Comment:
Is this safe with components-v1 ?

Reply:
Yes, I think. load-component-objects checks for v1, and if so, returns the component untouched. Is there any other problem you see?

Reply:
I was concern because there were no check for v2, but if it's done in load-component-objects, then is ok